features/shipping/viewing_shipping_methods/viewing_available_shipping_methods_based_on_channel.feature

```diff
@@ -0,0 +1,54 @@
+@viewing_shipping_methods
+Feature: Viewing available shipping methods based on channel
+    In order to only see applicable shipping methods
+    As a Visitor or Shop User
```

Comment:
Do these scenarios differ for Visitors and Shop Users? If they do, cannot they be in two separate files? If they do not, do we need to check them for both of these actors?

Reply:
These checks 2 situations - which shipping methods see Shop User and which Visitor during the cart checkout. I think We can leave them in this one file, becouse the only difference for them is one step:
`And I am a logged in customer`